statsmodels/iolib/tests/test_summary2.py

```diff
@@ -6,7 +6,7 @@
 
 from statsmodels.iolib.summary2 import summary_col
 from statsmodels.regression.linear_model import OLS, add_constant
-
+from statsmodels.regression.mixed_linear_model import MixedLM
```

Comment:
remove this again

There should not be any changes to this module

Reply:
Crap, sorry I missed it. It's done.